src/mbgl/geometry/feature_index.cpp

```diff
@@ -18,16 +18,17 @@
 namespace mbgl {
 
 FeatureIndex::FeatureIndex()
-    : grid(util::EXTENT, 16, 0) {
+    : grid(util::EXTENT, util::EXTENT, util::EXTENT >> 5) {
```

Comment:
Why `>> 5`?

Reply:
Uh, I'm not sure -- that's dividing it into a 32x32 grid, when it was a 16x16 grid before. Both cell sizes probably perform pretty similarly, but I'll revert to 16x16 and get rid of the confusing bit shift operator.